src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -59,169 +50,44 @@ const InputPassword: UISchemaElementComponent<UISchemaElementComponentWithValida
     focus,
     parserOptions,
     noTranslate,
-    showAsterisk,
-    dir,
   } = uischema;
   const {
     attributes,
     inputMeta: { name, required },
   } = uischema.options;
+  const { autocomplete } = attributes || {};
   const label = getTranslation(translations, 'label');
   const hint = getTranslation(translations, 'hint');
   const explain = getTranslation(translations, 'bottomExplain');
-  const optionalLabel = getTranslation(translations, 'optionalLabel');
   const focusRef = useAutoFocus<HTMLInputElement>(focus);
   const parsedExplainContent = useHtmlContentParser(explain, parserOptions);
-  const hasErrors = typeof errors !== 'undefined';
-  // TODO: OKTA-569647 - refactor logic
-  const hintId = hint && `${name}-hint`;
-  const explainId = explain && `${name}-explain`;
-  const ariaDescribedByIds = [describedByIds, hintId, explainId].filter(Boolean).join(' ')
-    || undefined;
-  const [showPassword, setShowPassword] = useState<boolean>(false);
-  const showPasswordTimeoutRef = useRef<number | undefined>();
-  const tokens = useOdysseyDesignTokens();
-
-  const handleClickShowPassword = () => {
-    setShowPassword(!showPassword);
-
-    if (showPasswordTimeoutRef.current) {
-      window.clearTimeout(showPasswordTimeoutRef.current);
-    }
-    // If the new value of showPassword is being set to true, set a 30-second timeout to auto-hide the password
-    // See: https://github.com/okta/okta-signin-widget#featuresshowpasswordtoggleonsigninpage
-    if (!showPassword) {
-      showPasswordTimeoutRef.current = window.setTimeout(() => {
-        setShowPassword(false);
-      }, 30000);
-    }
-  };
```

Comment:
Since we cannot pass in a handler for the toggle password function we will lose the 30-second auto-hide functionality but since this behavior is only [documented](https://github.com/okta/okta-signin-widget#featuresshowpasswordtoggleonsigninpage) with the `showPasswordToggleOnSignInPage` I thought it would be okay to deprecate it alongside the FF deprecation? The deprecation is planned for SIW 8.x